frontend/src/test/scala/bloop/bsp/BspBaseSuite.scala

```diff
@@ -606,6 +606,31 @@ abstract class BspBaseSuite extends BaseSuite with BspClientTest {
     }
   }
 
+  def loadBspMetalsBuildFromResources(
+      buildName: String,
+      workspace: AbsolutePath,
+      logger: RecordingLogger,
+      bspClientName: String,
+      bloopExtraParams: BloopExtraBuildParams = BloopExtraBuildParams.empty
```

Comment:
You can just add it to the method below, should work fine

Reply:
done!